releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -1,6 +1,7 @@
 ## Azure App Configuration Push Extension release notes
-Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version of the Azure App Configuration task will print to the console when the task is executed: ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
+Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information will print to the console when the task is executed:
+![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)

```

Comment:
>pictures/AzureDevOpsPushExtensionVersionSample.PNG [](start = 10, length = 50)

Should we update the extension and image to show the preview tag in the version?

Reply:
The "preview" tag is actually not part of the version on the ADO marketplace page. This is for illustration anyway. We probably won't update the image even when we release newer versions in the future.